plugins/napatech/plugin.c

```diff
@@ -21,7 +21,7 @@
 #include "decode.h"
 #include "source-napatech.h"
 #include "runmode-napatech.h"
-#include "util-device.h"
+#include "util-device-private.h"
```

Comment:
This was the safe way. Ideally someone who can build napatech should convert it to the opaque API.